pkg/crc/config/types.go

```diff
@@ -39,6 +39,10 @@ func (v SettingValue) AsInt() int {
 	return cast.ToInt(v.Value)
 }
 
+func (v SettingValue) AsUInt() uint {
+	return cast.ToUint(v.Value)
+}
```

Comment:
`AsInt()` may be good enough instead of adding a new helper?

Reply:
@cfergeau int can take nagative value as well so better to use `uint` just to safer side.

Reply:
I know, but most go code does not really care, see https://pkg.go.dev/net#LookupPort for example.

Reply:
https://pkg.go.dev/net/netip#AddrPortFrom uses uint16, not uint.

Reply:
`LookupPort` is returning the port number `AddrPortForm` is more accurate, I can use uint16 if that makes it more accurate.

Reply:
I'd be fine with just `int`, so pick whatever you prefer :)

Reply:
I am not going to change it in that case.